alerts/wildfly/README.md

```diff
@@ -0,0 +1,39 @@
+# Alert Policies for Wildfly in the Ops Agent
+
+## Server Errors
+
+If 5 or more server errors are happening over a 1 minute interval could be an indication that something is programatically incorrect or could use further exploration into why the server errors are occurring.
+
+## Waiting Requests
+
+Waiting requests could be an indication of high CPU usage or perhaps slow disk I/O. It would be recommended to use some kind of CPU utilization tool like top to dig deeper.
```

Comment:
Could we indicate the threshold that's used in the alert policy? For example, "If X or more requests are queued for more than Y minutes, then the alert policy is triggered"